integration/single-node-slug-deploy/check.go

```diff
@@ -62,6 +163,29 @@ func main() {
 	}
 }
 
+func loadGPG(workdir string) error {
+	importpub := exec.Command("gpg", "--no-default-keyring", "--keyring", path.Join(workdir, "pubring.gpg"), "--secret-keyring", path.Join(workdir, "secring.gpg"), "--import")
```

Comment:
good point, I'll add the rings to our repo instead of creating them at test time